src/EmployerRequestApprenticeTraining/SFA.DAS.EmployerRequestApprenticeTraining.Api/Controllers/StandardsController.cs

```diff
@@ -0,0 +1,46 @@
+﻿using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.EmployerRequestApprenticeTraining.Application.Queries.GetStandard;
+using SFA.DAS.SharedOuterApi.Models;
+using System;
+using System.Net;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.EmployerRequestApprenticeTraining.Api.Controllers
+{
+    [ApiController]
+    [Route("[controller]/")]
+    public class StandardsController : Controller
+    {
+        private readonly IMediator _mediator;
+        private readonly ILogger<StandardsController> _logger;
+
+        public StandardsController(IMediator mediator, ILogger<StandardsController> logger)
+        {
+            _mediator = mediator;
+            _logger = logger;
+        }
+
+        [HttpGet("{standardId}")]
+        public async Task<IActionResult> Get(string standardId)
+        {
+            try
+            {
+                var result = await _mediator.Send(new GetStandardQuery { StandardId = standardId });
+
+                if (result.Standard != null)
+                {
+                    return Ok(result.Standard);
+                }
+
+                return NotFound();
+            }
+            catch (Exception e)
+            {
+                _logger.LogError(e, $"Error attempting to retrieve standard with StandardId: {standardId}");
```

Comment:
SonarCloud has reported a security issue logging the standardId beacuse it is deeemed unsafe

Reply:
I have accepted this as a won't fix for now